doc/data-warehouse-what-and-why.md

```diff
@@ -0,0 +1,263 @@
+publishing, updating and archiving/retiring their content daily. Diligent
+management of content implies a steady pace of updates and publications while
+keeping the total number of content items in a sensible number.
+
+For example, if an organization is publishing at a very high rate for an
+extended period of time, it could be a signal that they need to improve the way
+they archive or retire existing content.
+
+To get these insights about content publication we can't focus on metrics for a
+single day, we need to understand how those metrics evolved **over time**.
+
+Let's explore the example:
+
+* **Content Items published by an organization in one day**
+
+We have two organizations: `A` and `B`. In the last day, `Organisation A` has
+`550` live Content Items and `Organisation B` has `400`.
+
+With this information, it is hard to see if those organizations are effectively
+managing their content because we only have a data points for a single day.
+
+![without metrics over time][figure-1-no-trends]
+
+* **Content Items published by an organization over time***
+
+If we track the number of content items **over time**, then we might observe
+that `Organisation A` is decreasing the number of Content Items daily, and
+`Organisation B` has created too many content items in the last year.
+
+We can conclude that `organization b` would need to take action to improve the
+way they manage content over time: they can make an informed decision based on
+data about the way they manage content.
+
+![with metrics over time][figure-2-with-trends]
+
+### Example 2: Enforcing good practices
+
+Data is not only to help departments to manage their content over time; it can
+also help to provide evidence about compliance to best practices and guidelines.
+
+For example, we discourage departments from publishing content in PDF format.
+There are many reasons for it, but having some metrics to prove it would
+decrease the distance between the GOV.UK guidelines and department practices
+when publishing content.
+
+In the following example, we can see how there is a relationship between the
+number of PDFs published by `Organisation A` and the `satisfaction score`.
+
+![enforcing good practices][figure-3-comparisons]
+
+### Example 3: Predictions
+
+User interactions with content on GOV.UK can sometimes follow seasonal patterns.
+If on a particular week or month of the year a set of pages receive a spike on
+the number of visits, it is possible those pages will receive a similar number
+of visits the following year during the same period.
+
+Let's explore the following example:
+
+As per the spike of visits in June last year, it is very likely that
+`Organisation A` expects a spike in page views for a content item, so they might
+consider reviewing in advance the content, so it is accurate and up to date.
+
+![predictions][figure-4-predictions]
+
+# Why a Data Warehouse
+
+## State of the art
```

Comment:
This section's structure and purpose make much more sense now! 

But the title is confusing, how about `Considerations for building new application`? 
"State of the art" doesn't make sense on its own and is closer to "the cutting edge" in meaning.